pkg/resource/testdata/TestStatefulSetBuilder/default_secure.golden

```diff
@@ -74,6 +74,11 @@ spec:
             scheme: HTTPS
           initialDelaySeconds: 10
           periodSeconds: 5
+        lifecycle:
+          preStop:
```

Comment:
we need to add terminationGracePeriodSeconds and set it to 60 seconds here

Reply:
@chrislovecnm  it is already in there, but at the pod spec level not container level ... see https://github.com/cockroachdb/cockroach-operator/blob/master/pkg/resource/statefulset.go#L198